logstash-core-event-java/lib/logstash/string_interpolation.rb

```diff
@@ -0,0 +1,18 @@
+# encoding: utf-8
+
+module LogStash
+  module StringInterpolation
+    extend self
+
+    # clear the global compiled templates cache
+    def clear_cache
+      Java::ComLogstash::StringInterpolation.get_instance.clear_cache;
```

Comment:
I believe this is the preferred way of referencing Java classes, the other notations AFAIU is only for  top level Java packages java, javax, org, and com per https://github.com/jruby/jruby/wiki/CallingJavaFromJRuby#referencing-java-classes-using-full-qualified-class-name